tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -66,3 +66,86 @@ def __str__(self):
     def _get_error_message(self):
         """Return the error message."""
         return self.error_message
+
+class DataFileSummary(models.Model):
+    """Aggregates information about a parsed file."""
+
+    class Status(models.TextChoices):
+        """Enum for status of parsed file."""
+
+        PENDING = "Pending"  # file has been uploaded, but not validated
+        ACCEPTED = "Accepted"
+        ACCEPTED_WITH_ERRORS = "Accepted with Errors"
+        REJECTED = "Rejected"
+
+    status = models.CharField(
+        max_length=50,
+        choices=Status.choices,
+        default=Status.PENDING,
+    )
+
+    datafile = models.ForeignKey(DataFile, on_delete=models.CASCADE)
```

Comment:
TODO: update to more restrictive version:
https://github.com/raft-tech/TANF-app/blob/0193d9f4e2d8c4573639f841c064197a4e778b0c/tdrs-backend/tdpservice/parsers/models.py#L87

Reply:
Can't actually condense this into previous migration since ~5 other migrations exist between present state and introducing this from merges into this branch. This will get resolved as part of #2622